go.mod

```diff
@@ -5,5 +5,8 @@ go 1.13
 require (
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/golang/protobuf v1.3.2
+	github.com/google/go-cmp v0.4.0 // indirect
```

Comment:
is this stuff coming from something VSCode told you to install (this happened to George)? can you please revert these files?